docs/dev/cve.md

```
@@ -1,69 +1,68 @@
-# Handling CVEs in the project
+   # Handling CVEs in the project
```

markdownlint-cli2 (0.22.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)